src/centos/scripts/start-marklogic.sh

```diff
@@ -13,6 +13,41 @@
 
 cd ~
 
+################################################################
+# Read in ENV values
+################################################################
+
+# If an ENV value exists in a list Append it the /etc/marklogic.conf file 
+if [[ -z $OVERWRITE_ML_CONF ]] || [[ "$OVERWRITE_ML_CONF" = "false" ]] ; then
```

Comment:
I created this option as an opt in so users need to pass in `OVERWRITE_ML_CONF` in order for the code to execute. 

I did this because the code will overwrite the `/etc/marklogic.conf` file on the image. It needs to do this because of the following situations: 

I wanted to prevent a case where we keep appending repeated information to the file and I also didn't want to add the logic to determine if the user wants us to overwrite a specific variable in the file but leave the rest since I viewed this as a stretch case.